### PR TITLE
Lycan Chain Added.

### PR DESCRIPTION
**Pull request by Lycan-Chain**
## Changes:
- Lycan Chain Added.

### CODE REVIEW

**Comment by LukaszRozmej** [comment]
@Lycan-Chain are you planning to have some built-in config file too? Would be more useful if you want default support.

**Comment by Lycan-Chain** [comment]
We have added the config file.

> @Lycan-Chain are you planning to have some built-in config file too? Would be more useful if you want default support.

**Comment by LukaszRozmej** [comment]
@kamilchodola can we confirm it syncs?

**Comment by kamilchodola** [comment]
@Lycan-Chain @LukaszRozmej 
[lycan.logs.txt](https://github.com/NethermindEth/nethermind/files/9887847/lycan.logs.txt)

I tried to sync a node using this chainspec and config - was waiting for about 20h and never been able to find any peer to start syncing.

**Comment by kamilchodola** [comment]
@chainsensedhiraj 
Please check above comment

File: src/Nethermind/Nethermind.Runner/configs/lycan.cfg [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "ChainSpecPath": "chainspec/lycan.json",
+    "BaseDbPath": "nethermind_db/lycan",
+    "LogFileName": "logs/lycan.logs.txt"
+  },
+  "Network": {
+    "DiscoveryPort": 30300,
+    "P2PPort": 30300,
+    "ActivePeersMaxCount": 100
+  },
+  "JsonRpc": {
```
Comment (flcl42) [inline src/Nethermind/Nethermind.Runner/configs/lycan.cfg:15]: It will most likely ask for at least Engine to be enabled

**Comment by LukaszRozmej** [comment]
Not enough engagement from Lycan chain, we can reopen if needed.

**Pull request closed without merging**